### ISSUE TITLE
Additonal Dragon Materials. 

### ISSUE THREAD

**Issue by MrLaughingJack22** [AlexModGuy/Ice-and-Fire-Suggestions#81]
So, a idea I had is for the dragon to be able to drop it's horns, teeth and talons.

- Horns; The horns could be another crafting material to use in the top left and right corners of the helmet recipe for the dragon armor, as well as the Dragon Horn itself.

- Teeth; Could be used to make a some more weapons, and as well as decoration to set on your house.

- Talons; These could be used as a crafting material for the tools, or maybe a knife to scrape more materials off of dragon.

**Comment by LordDecay26** [comment]
i agree with the dragon horn idea! imagine being able to add dragon horns to regular helmets in order to have what is essentially a viking helmet! it would be a aesthetic thing but it would be soooo cool!
